core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -320,9 +320,10 @@ object TopicCommand extends Logging {
         val allConfigs = adminClient.describeConfigs(topics.map(new ConfigResource(Type.TOPIC, _)).asJavaCollection).values()
         val liveBrokers = adminClient.describeCluster().nodes().get().asScala.map(_.id())
         val topicDescriptions = adminClient.describeTopics(topics.asJavaCollection).all().get().values().asScala
+          .toSeq.sortBy(td => td.name())
```

Comment:
Could you add comment for this `sortBy`?

Reply:
I have added a comment.